libs/ui/autocomplete/tsconfig.schematics.json

```diff
@@ -0,0 +1,37 @@
+{
+  "compilerOptions": {
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
+    "baseUrl": ".",
+    "lib": [
+      "es2018",
+      "dom"
+    ],
+    "declaration": true,
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "noEmitOnError": true,
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "noUnusedParameters": true,
+    "noUnusedLocals": true,
+    "rootDir": "schematics",
+    "outDir": "../../../dist/libs/ui/autocomplete/schematics",
+    "skipDefaultLibCheck": true,
+    "skipLibCheck": true,
+    "sourceMap": true,
+    "strictNullChecks": true,
+    "target": "es6",
+    "types": [
+      "jest",
```

Comment:
probably moot point if we are able to combine tsconfigs

Reply:
I don't _think_ Jest is required there... In Angular schematics docs, this is list there, so I just copy over.
Would be nice if these two combined, but it might require some deep digging as it doesn't seem to be Angular convention for now. Could be one of those nice-to-have features.

Reply:
Have we tried combining and it failed? Or just haven't tried?

Reply:
I tried it at beginning. With the `rootDir` and `outDir` being different, it was kinda hard to use one to make both working.